posthog/queries/trends.py

```diff
@@ -246,6 +246,28 @@ def get_aggregate_total(query: QuerySet, entity: Entity) -> int:
     return entity_total
 
 
+def aggregate_cohort_filter(
```

Comment:
This name is confusing for me since it's returning an int not filtering a queryset.

Would it make sense to rename this to `get_aggregate_cohort_total`?

Reply:
done